src/main/resources/liquibase/crop_changelog/19_5_1.xml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+	
+	<changeSet author="rowena" id="v19.5.1-1">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) > 0
```

Comment:
Hi @mchabito, @rvtulod - my concern is it's only checking and updating usage of the "dupe" location in germplsm table. However, there are several other places where a location could be referred:
- atributs.alocn
- instln.ulocn (not sure)
 - names.nlocn
- study variables (mainly nd_geolocation but also possible in projectprop,phenotype when a variable has location data type..)

I think we should be checking/updating usage of those too before we delete any location to avoid orphan records.